light-client/src/contracts.rs

```diff
@@ -40,6 +38,6 @@ pub fn light_store_contains_block_within_trusting_period(
 //     target_height: Height,
 // ) -> bool {
 //     light_store
-//         .all(VerifiedStatus::Verified)
+//         .all(Status::Verified)
 //         .all(|lb| lb.height() < target_height)
 // }
```

Comment:
Why is this commented out method in here?

Reply:
It's here because it is needed by the (commented-out) precondition of `LightClient::verify_to_target`. It's disabled because it used to cause problems before we had a `Clock` component. I just didn't get around to re-enabling the precondition since we have that. Thanks for reminding me to look into that, tracked in #382.